openedx/core/djangoapps/external_user_ids/apps.py

```diff
@@ -0,0 +1,16 @@
+"""
+External User ID Application Configuration
+"""
+
+
+from django.apps import AppConfig
+
+
+class ExternalUserIDConfig(AppConfig):
+    """
+    Default configuration for the "openedx.core.djangoapps.credit" Django application.
+    """
+    name = 'openedx.core.djangoapps.external_user_ids'
+
+    def ready(self):
+        from . import signals  # pylint: disable=unused-variable
```

Comment:
I don't want to distract from the overall review but I was curious... what is the significance of this line? Why do we need it? I think my next learning time might need to be dedicated to signals... :)

Reply:
When the app is loaded and ready is attaches the signal receivers in signals.py